public/app/plugins/panel/candlestick/panelcfg.cue

```diff
@@ -14,23 +14,55 @@
 
 package grafanaplugin
 
+import (
+	"github.com/grafana/grafana/packages/grafana-schema/src/common"
+)
+
 composableKinds: PanelCfg: {
+	maturity: "experimental"
+
 	lineage: {
-		seqs: [
-			{
-				schemas: [
-					{
-						Options: {
-							// anything for now
-							...
-						} @cuetsy(kind="interface")
-						FieldConfig: {
-							// anything for now
-							...
-						} @cuetsy(kind="interface")
-					},
-				]
-			},
-		]
+
+		seqs: [{
```

Comment:
Did this structure change? Particularly from:

`schemas: [{
			version: [0, 0]
			schema: {`